test/unit/machine-learning/machine-learning.spec.ts

```diff
@@ -123,6 +138,67 @@ describe('MachineLearning', () => {
     },
   };
 
+  const MODEL_RESPONSE5: any = {
+    name: 'projects/test-project/models/3456789',
+    createTime: '2020-02-07T23:45:23.288047Z',
+    updateTime: '2020-02-08T23:45:23.288047Z',
+    etag: 'etag345',
+    modelHash: 'modelHash345',
+    displayName: 'model_3',
+    tags: ['tag_3', 'tag_4'],
+    state: { published: true },
+    tfliteModel: {
+      foo: true,
+      sizeBytes: 22200222,
+    },
+  };
+
+  const MODEL_RESPONSE6: any = {
+    name: 'projects/test-project/models/3456789',
+    createTime: '2020-02-07T23:45:23.288047Z',
+    updateTime: '2020-02-08T23:45:23.288047Z',
+    etag: 'etag345',
+    modelHash: 'modelHash345',
+    displayName: 'model_3',
+    tags: ['tag_3', 'tag_4'],
+    state: { published: true },
+    coremlModel: {
+      gcsCoremlUri: 'gs://test-project-bucket/Firebase/ML/Models/model6.mlmodel',
+      sizeBytes: 33300333,
+    },
+  };
+
+  const MODEL_RESPONSE7: any = {
+    name: 'projects/test-project/models/3456789',
+    createTime: '2020-02-07T23:45:23.288047Z',
+    updateTime: '2020-02-08T23:45:23.288047Z',
+    etag: 'etag345',
+    modelHash: 'modelHash345',
+    displayName: 'model_3',
+    tags: ['tag_3', 'tag_4'],
+    state: { published: true },
+    coremlModel: {
+      managedUpload: true,
+      sizeBytes: 33300333,
+    },
+  };
+
+  const MODEL_RESPONSE8: any = {
```

Comment:
These names are becoming quite hard to follow. It was manageable when it was just 2-3, but now we are up to 8. Going through the test cases, it requires quite a bit of effort to follow and understand the logic. Highly recommend we give more descriptive names to these, so when they appear in the test cases it makes sense.
Suggestions: `COREML_MODEL_WITH_UNKNOWN_FIELD`, `COREML_MODEL_WITH_MANAGED_UPLOAD`, `COREML_MODEL_WITH_GCS_URI` and so on.

Reply:
done